File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    float dollars;

    do
    {
        dollars = get_float("Change owed: ");
    }
    while (dollars < 0);

    int cents = round(dollars * 100); //convert dollars to cents rounded to nearest penny

    int minCoins = 0;
    minCoins += cents / 25; //Add quarters
    cents = cents % 25; //leftover change

    minCoins += cents / 10; //Add dimes
    cents = cents % 10; //leftover change

    minCoins += cents / 5; //Add nickels
    cents = cents % 5; //leftover change

    minCoins += cents; //Add pennies

    printf("%i\n", minCoins);
}

File: pset6/cash/cash.py
from cs50 import get_float

while True:
    change = get_float("Change owed: ")
    if change > 0:
        break

coins = 0
cents = int(change * 100)

# Add the number of quarters, dimes, and nickels to coins
for coin in [25, 10, 5]:
    coins += cents // coin
    cents %= coin
# Add pennies
coins += cents

print(coins)